app/assets/stylesheets/tile.css

```diff
@@ -14,6 +14,18 @@
     .tile-description {
         width: 300px;
     }
+
+    .tile-title {
+        width: 300px;
+    }
+}
+
+
+@media screen and (max-width : 799px) and (min-width: 481px) {
```

Comment:
What do we have this block for? The condition selects a subset of elements of the `@media`'s condition above and the CSS values are the same. Remove?